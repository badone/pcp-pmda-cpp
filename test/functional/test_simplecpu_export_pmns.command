./pmdasimplecpu --export-pmns | grep -Eve 'cpu[1-9]'
