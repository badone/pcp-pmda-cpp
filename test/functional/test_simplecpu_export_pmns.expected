
simplecpu {
    ticks
}

simplecpu.ticks {
    total    SIMPLECPU:0:0
    cpu0     SIMPLECPU:0:1
}

